src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3111,44 +2135,16 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     // - eventArgs: event for storing the current font information
     // Return Value:
     // - <none>
-    void TermControl::_FontInfoHandler(const IInspectable& /*sender*/, const FontInfoEventArgs& eventArgs)
+    void TermControl::_FontInfoHandler(const IInspectable& /*sender*/,
+                                       const FontInfoEventArgs& eventArgs)
     {
         eventArgs.FontSize(CharacterDimensions());
-        eventArgs.FontFace(_actualFont.GetFaceName());
+        eventArgs.FontFace(_core->GetFont().GetFaceName());

```

Comment:
if font is a copy here, we'll want to save it. if it's a pointer, that's ok. if a reference, that can't go across the [future, winrt] boundary ;P

Reply:
It's a copy for now. I'm imagining this will get projected as `String FontName{ get; }`, etc in the future.

Reply:
Yeah... since it's a copy, don't call it two times back-to-back two lines apart :wink: